apps/docs/docs/reference/ui/molecules/InputField.mdx

```diff
@@ -0,0 +1,208 @@
+---
+description: InputFields are text fields used to get user inputs.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Input___77d0e7a8079b7d310e8223dd9e1994f6.png
+---
+
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+import { TokenTable, TokenRow } from '@site/src/components/Tokens'
+
+# InputField
+
+InputField wraps an [Input](/reference/ui/atoms/Input) and its corresponding [Label](/reference/ui/atoms/Label).
+
+## Overview
+
+```tsx live
+<div style={{ display: 'flex', flexDirection: 'column', rowGap: '22px' }}>
```

Comment:
is the `overviewSection` class applied here?

Reply:
non, but `overviewSectionColumn` yes. Lemme update it! Thanksss